rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java

```diff
@@ -20,16 +12,27 @@
 import org.ow2.proactive.resourcemanager.utils.OperatingSystem;
 import org.ow2.proactive.utils.Tools;
 
+import java.io.IOException;
+import java.security.KeyException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
 
 public class LocalInfrastructure extends InfrastructureManager {
 
+    public static final int DEFAULT_NODE_NUMBER = 4;
```

Comment:
The same number as used for SchedulerStarter could maybe be used by default?

See `RMStarter#DEFAULT_NODES_NUMBER`:

```
Math.max(2, Runtime.getRuntime().availableProcessors() - 1)
```